mkdocs-base.yaml

```
@@ -0,0 +1,70 @@
+site_name: QX simulator
+site_dir: public
+repo_url: https://github.com/QuTech-Delft/qx-simulator
+use_directory_urls: false
+
+nav:
+  - Home: index.md
+  - User Guide:
+      - User Guide: user-guide/user-guide.md
+      - Command Line: user-guide/command-line.md
+      - Python: user-guide/python.md
+  - Developer Guide:
+      - Dev Guide: dev-guide/dev-guide.md
+      - C++: dev-guide/cpp.md
+      - Python: dev-guide/python.md
+  - API:
+      - C++: api/cpp.md
+      - Python: api/python.md
```

Do we want unique names for the pages?
```suggestion
      - Python: user-guide/python-user-guide.md
  - Developer Guide:
      - Dev Guide: dev-guide/dev-guide.md
      - C++: dev-guide/cpp-dev-guide/.md
      - Python: dev-guide/python-dev-guide/.md
  - API:
      - C++: api/cpp-api.md
      - Python: api/python-api.md
```

---

OK, we can do.